Contentful.NET.PCL/Contentful.NET.PCL.csproj

```diff
@@ -37,120 +37,6 @@
     <WarningLevel>4</WarningLevel>
   </PropertyGroup>
   <ItemGroup>
```

Comment:
This entire file should have been deleted?

Reply:
Ah, yes. I only removed it from the sln. Fixed now.